src/PKSim.Infrastructure/ORM/Repositories/ParametersInContainerRepository.cs

```diff
@@ -43,19 +30,23 @@ public ParameterMetaData ParameterMetaDataFor(IParameter parameter)
          if (parameter == null)
             return null;
 
-         return ParameterMetaDataFor(_entityPathResolver.PathFor(parameter));
-      }
+         if (parameter.IsDistributed())
```

Comment:
I thought I was clever with this implementation but this does not work. We need to know CM or PVV depending on the parameter type... problem is that we only have the parameter in the command